test/fixtures/api/responses/get-transactions.json

```diff
@@ -3,8 +3,16 @@
     "type": "payment",
     "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
     "sequence": 4,
-    "id": "4C37C92576DEB000D13B07F4D3F99F968BD86B6B83A840BEFFB2BFC8A042A81B",
+    "id": "99404A34E8170319521223A6C604AF48B9F1E3000C377E6141F9A1BF60B0B865",
     "specification": {
+      "memos": [
+        {
```

Comment:
ok
BTW, as I understand, that one
https://github.com/ripple/ripple-client/blob/develop/src/js/tabs/send.controller.js#L1121
should be 
`tx.addMemo('client', 'plain/text', 'rt' + $scope.version);`